GPU/Workflow/src/O2GPUDPLDisplay.cxx

```diff
@@ -94,12 +94,6 @@ void O2GPUDPLDisplaySpec::run(ProcessingContext& pc)
   GPUTrackingInOutPointers ptrs;
   auto tmpContainer = GPUWorkflowHelper::fillIOPtr(ptrs, recoData, mUseMC, &(mConfig->configCalib), mClMask, mTrkMask, mTrkMask);
```

Comment:
@martenole : After this PR, you can use the `GPUWorkflowHelper::fillIOPtr` here to get the input data in the TRD Tracking workflow. For your case, you can put nullptr as calibObject, since it is not needed. And use GID::TRD for the cluster mask (since there is no tracklet mask, trd clusters means it will fetch trd tracklets), and then request the TPC tracks and the TPCITS tracks.

It will automatically fill the ioPtr, and it will call `createTracks` to resolve the duplicate tracks. All TPC tracks with `ptrs.tpcLinkITS != -1` are present also as ITS-TPC track.

You cannot remove the trd::RecoInputContainer, since I use it internally. But I am not using the part with the tpc tracks and tpc-its tracks in the RecoInputContainer, only the trd tracklets and trigger records. So after you switch, you can remove the track part in the RecoInputContainer.